### PR TITLE
Adding support to configure RV service component-samples using properties file.

### PR DESCRIPTION
**Pull request by Darshini-Parikh93**
- The application can be configured using properties.
- Reading RV DB and Tomcat context parameters using Constants
and Configuration Loader classes.

TO-DO:
-Create README for service/component-samples/rv

Signed-off-by: Darshini Parikh <darshini.k.parikh@intel.com>

### CODE REVIEW

**Review by johneasterday — DISMISSED** [review]
Minor quibble - RvAppConstants are not really constants (the name is but not the values). Something like RvAppSettings or the like might be better.

File: service/component-samples/rv/src/main/java/org/fido/iot/sample/RvAppConstants.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  public static final String DB_PASSWORD = "";
+
+  public static final String WEB_PATH = "/fido/100/msg";
+  public static final String DB_PATH =
+      Path.of(System.getProperty("user.dir"), "target", "data", "rvs").toString();
+  public static final String SERVER_PATH =
+      Path.of(System.getProperty("user.dir"), "target", "tomcat").toString();
+
+  public static final String CATALINA_HOME = "catalina.home";
+  public static final String DB_USER_KEY = "db.user";
```
Comment (prateekcr) [inline service/component-samples/rv/src/main/java/org/fido/iot/sample/RvAppConstants.java:20]: Please update the properties separator to '-', '_', from '.'. Dot separators are problematic while setting properties as environment variables.

**Comment by Darshini-Parikh93** [comment]
> Minor quibble - RvAppConstants are not really constants (the name is but not the values). Something like RvAppSettings or the like might be better.

Done

File: service/component-samples/rv/src/main/java/org/fido/iot/sample/RvAppConstants.java (hunk 1 above)
Comment (Darshini-Parikh93) [reply service/component-samples/rv/src/main/java/org/fido/iot/sample/RvAppConstants.java]: Done

**Review by prateekcr — APPROVED** [review]

**Review by johneasterday — APPROVED** [review]

**Pull request merged**